code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -106,6 +106,10 @@
 	name = "Fireball"
 	spell_type = /obj/effect/proc_holder/spell/aimed/fireball
 
+/datum/spellbook_entry/grease
+	name = "Grease"
+	spell_type = /obj/effect/proc_holder/spell/aimed/grease

```

Comment:
if wizard sandals DON'T make you immune to grease, I think we should lower grease to being a 1 point spell (why take grease when you can just take fireball, which also knocks down its victims, is robeless, and deals damage too?)

I eagerly await the day when a wizard with The Traps! x5 and Grease x5 manages to greentext (actually, would the long slips from the greased tiles slip people into the traps from the The Traps! spell? that might actually be a really cool combo)

Reply:
is this a 2 cost spell? you know lube tiles can be removed by just crowbaring the tile

Reply:
lowered to 1